### PR TITLE
Add instructions to install istio 1.6

### PR DESCRIPTION
**Pull request by tbruno**
This PR adds instructions to use Anypoint Service Mesh with Istio 1.6.x

### CODE REVIEW

**Review by dosten — DISMISSED** [review]
LGTM!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 1]
```diff
@@ -83,11 +83,74 @@ In your network, you might need to whitelist the hostnames and ports of external
 [[install_Istio]]
 == Installing Istio
```
Comment (kbostrom) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]:
```suggestion
== Install Istio
```

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 2]
```diff
@@ -83,11 +83,74 @@ In your network, you might need to whitelist the hostnames and ports of external
...
 
-You can install Istio 1.4.x or 1.5.x in your environment to support Anypoint Service Mesh. However, the Istio configuration for Anypoint Service Mesh differs based on the Istio version that you installed.
+You can install Istio 1.6.x, 1.5.x, or 1.4.x in your environment to support Anypoint Service Mesh. However, the Istio configuration for Anypoint Service Mesh differs based on the Istio version that you installed.
```
Comment (kbostrom) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc:86]: Are we documenting installation or configuration?
Seems like the latter:
```suggestion
Although you can install Istio 1.6.x, 1.5.x, or 1.4.x in your environment to support Anypoint Service Mesh, the Istio configuration for Anypoint Service Mesh depends on the Istio version that you installed.
```

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 3]
```diff
@@ -83,11 +83,74 @@ In your network, you might need to whitelist the hostnames and ports of external
...
+
+=== Install Istio 1.6.x
```
Comment (kbostrom) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]:
```suggestion
=== Install and Configure Istio 1.6.x
```

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 2 above)
Comment (kbostrom) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc:86]: Any prerequisites that apply to all three installations (=== Prerequisites)?

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 4]
```diff
@@ -83,11 +83,74 @@ In your network, you might need to whitelist the hostnames and ports of external
...
+
+Anypoint Service Mesh supports Istio versions `1.6.0` through `1.6.8`. To install and configure Istio 1.6.x for Anypoint Service Mesh:
```
Comment (kbostrom) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]:
```suggestion
Anypoint Service Mesh supports Istio 1.6.x versions 1.6.0 through 1.6.8. To install and configure Istio 1.6.x for Anypoint Service Mesh:
```

Comment (kbostrom) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Consider whether there are prereqs for installing 1.6.x. 
If so, document them following the first sentence above (==== Prerequisites) and then use the subtitle ==== Steps to transition from prereqs to steps.

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 5]
```diff
@@ -83,11 +83,74 @@ In your network, you might need to whitelist the hostnames and ports of external
...
+
+. Download Istio using the https://istio.io/docs/setup/[Istio documentation].
+. Install Istio with the following configurations:
```
Comment (kbostrom) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: How does one "install with configurations"? Doesn't one first install and then configure?
1. Download
2. Install
3. Enable flag
4. Enable telemetry
5. Install manifest? What is a manifest? It sounds like it is separate from Istio, which has already been installed. What is a full manifest vs. a not full manifest?

The topic titles here are incorrect if the task includes three tasks:
1. Download
2. Install
3. Configure

Review to ensure that the language used throughout this topic is precise.

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 6]
```diff
@@ -83,11 +83,74 @@ In your network, you might need to whitelist the hostnames and ports of external
...
 == Installing Istio
 
-You can install Istio 1.4.x or 1.5.x in your environment to support Anypoint Service Mesh. However, the Istio configuration for Anypoint Service Mesh differs based on the Istio version that you installed.
+You can install Istio 1.6.x, 1.5.x, or 1.4.x in your environment to support Anypoint Service Mesh. However, the Istio configuration for Anypoint Service Mesh differs based on the Istio version that you installed.
+
+=== Install Istio 1.6.x
+
+Anypoint Service Mesh supports Istio versions `1.6.0` through `1.6.8`. To install and configure Istio 1.6.x for Anypoint Service Mesh:
+
+. Download Istio using the https://istio.io/docs/setup/[Istio documentation].
```
Comment (kbostrom) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Downloading is not part of installing (and configuring). 
This looks like it should be a prerequisite, not the first step of installing.

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 7]
```diff
@@ -83,11 +83,74 @@ In your network, you might need to whitelist the hostnames and ports of external
...
+      enabled: true
+    v2:
+      enabled: false
+components:
+  citadel:
+    enabled: true
+  telemetry:
+    enabled: true
+----
+The following example illustrates a full manifest:
```
Comment (kbostrom) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc:124]: What is this? How does it relate to:
1. Installing
2. Enabling flag
3. Enabling telemetry

It's position indicates that it is an example of enabling telemetry.

Comment (kbostrom) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc:124]:
```suggestion
With the policy control flag and telemetry enabled, the configuration looks like this:
```
Where does "manifest" fit in?

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 8]
```diff
@@ -83,11 +83,74 @@ In your network, you might need to whitelist the hostnames and ports of external
...
+  meshConfig:
+    disablePolicyChecks: false
+  values:
+    telemetry:
+      v1:
+        enabled: true
+      v2:
+        enabled: false
+----  
+When using the full manifest, ensure that you choose the correct profile. In the example, the `default` profile is used.
```
Comment (kbostrom) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc:148]: So...doesn't configuration begin with specifying the correct profile? Profile is the first component of the install process, so it should be described first, shouldn't it?
1. Install
2. Specify profile
3. Enable policy control flag
4. Enable telemetry

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 9]
```diff
@@ -83,11 +83,74 @@ In your network, you might need to whitelist the hostnames and ports of external
...
+  values:
+    telemetry:
+      v1:
+        enabled: true
+      v2:
+        enabled: false
+----  
+When using the full manifest, ensure that you choose the correct profile. In the example, the `default` profile is used. 
+
+To install the manifest, run:
```
Comment (kbostrom) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc:150]: This is worded as if it is a whole separate task, if installing the manifest is not part of installing Istio.
Clarify.
If this is a step in the install task, it should be numbered and placed in its chronological location.

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 10]
```diff
@@ -83,11 +83,74 @@ In your network, you might need to whitelist the hostnames and ports of external
...
++
+`istioctl manifest apply -f <manifest-file.yaml>`
 
 === Install Istio 1.5.x
 
-Anypoint Service Mesh supports Istio versions `1.5.0` through `1.5.2`. To install and configure Istio 1.5.x for Anypoint Service Mesh:
+Anypoint Service Mesh supports Istio versions `1.5.0` through `1.5.9`. To install and configure Istio 1.5.x for Anypoint Service Mesh:
```
Comment (kbostrom) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: ditto same edits made to similar sentence for 1.6.x

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 11]
```diff
@@ -158,7 +221,7 @@ To install the manifest, run:
 
 === Install Istio 1.4.x
 
-Anypoint Service Mesh supports Istio versions `1.4.0` through `1.4.8`. To install and configure Istio 1.4.x for Anypoint Service Mesh:
+Anypoint Service Mesh supports Istio versions `1.4.0` through `1.4.10`. To install and configure Istio 1.4.x for Anypoint Service Mesh:
```
Comment (kbostrom) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: ditto

**Review by kbostrom — DISMISSED** [review]
Edited per scope of request

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc:86]: None! Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 4 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 5 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 6 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 7 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc:124]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 8 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc:148]: Left a question for the engineers. Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 9 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc:150]: Left a comment for the engineers. Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 10 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Nope the supported versions are as have been documented. Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 11 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 12]
```diff
@@ -81,16 +81,83 @@ In your network, you might need to whitelist the hostnames and ports of external
...
 
 [[install_Istio]]
-== Installing Istio
+== Download, Install, and Configure Istio
 
-You can install Istio 1.4.x or 1.5.x in your environment to support Anypoint Service Mesh. However, the Istio configuration for Anypoint Service Mesh differs based on the Istio version that you installed.
+You can install Istio 1.6.x, 1.5.x, or 1.4.x in your environment to support Anypoint Service Mesh. However, the Istio configuration for Anypoint Service Mesh differs based on the Istio version that you installed.
 
-=== Install Istio 1.5.x
+=== Task Prerequisites
```
Comment (kbostrom) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]:
```suggestion
=== Prerequisites
```

**Review by kbostrom — DISMISSED** [review]
Reapproved per scope of request

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 12 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

**Review by kbostrom — APPROVED** [review]
Reapproved

**Pull request merged**